python/GangaLHCb/test/GPI/TestBookkeeping.py

```diff
@@ -0,0 +1,108 @@
+""" Test bookkeeping interface"""
+from __future__ import absolute_import
+
+import os
+import os.path
+
+from Ganga.Utility.logging import getLogger
+from Ganga.testlib.mark import external
+from Ganga.testlib.GangaUnitTest import GangaUnitTest
+
+from Ganga.Core.exceptions import PluginError
+
+logger = getLogger(modulename=True)
+
+
+@external
+class TestBookkeeping(GangaUnitTest):
+    """Test the bookkeeping interface"""
+
+    def test_path_query(self):
+        """Test that a path query works"""
+        from Ganga.GPI import BKQuery, LHCbDataset
+        bkq = BKQuery(
+            dqflag=['OK', 'UNCHECKED'],
+            path="/LHCb/Collision16/Beam6500GeV-VeloClosed-MagDown/Real Data/91000000/RAW",
+            type="Path"
+        )
+        dataset = bkq.getDataset()
+
+        assert isinstance(dataset, LHCbDataset)
+        assert len(dataset) > 0
+
+    def test_date_query(self):
```

Comment:
Although having said that I cannot get his to work at all even with LHCbDirac `dirac-bookkeeping-get-files` on the command line so maybe there is an issue there?